scripts/release-binary.sh

```diff
@@ -23,30 +23,45 @@ export PATH=/usr/lib/llvm-8/bin:$PATH
 export CC=${CC:-clang}
 export CXX=${CXX:-clang++}
 
-# The bucket name to store proxy binary
+# The bucket name to store proxy binaries.
 DST="gs://istio-build/proxy"
 
+# Verify that we're building binaries on Ubuntu 16.04 (Xenial).
+CHECK=1
+
 function usage() {
   echo "$0
-    -d  The bucket name to store proxy binary (optional)"
+    -d  The bucket name to store proxy binary (optional).
+        By default, the GCS bucket is set to \"gs://istio-build/proxy\".
+        Use \"none\" to skip the GCS upload.
```

Comment:
i wonder if we should make the default 'none'?

Reply:
Yes, but right now `proxy-postsubmit` calls it without arguments, so it would break again. We can do it in subsequent change, though.